nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py

```diff
@@ -56,4 +56,11 @@ def get_metadata(self):
                 )
             )
         )
+
         return ecephys_metadata
+
+
+class SpikeGLXLFPInterface(BaseLFPExtractorInterface):
+    """Primary data interface class for converting the high-pass (ap) SpikeGLX format."""
```

Comment:
usually it's the low pass that's called "lfp"

Reply:
Whoops, that would left over from copying/pasting from the RecordingInterface. Fixed!